lib/kennel/importer.rb

```diff
@@ -21,6 +21,13 @@ def import(resource, id)
       data = @api.show(model.api_resource, id)
 
       id = data.fetch(:id) # keep native value
+
+      if resource == "slo"
+        # we ignore the 3 copy-pasted fields that track the primary threshold, so we need to sort imported thresholds
+        # or the primary information is lost
+        data[:thresholds].sort! { |t| t[:timeframe] == data[:timeframe] ? 0 : 1 }
```

Comment:
this does not work, when creating a slo datadog always picks the lowest one as primary

Reply:
ah nvm it should work with the slo change 🤞